book.toml

```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["Kat Anderson", "Benjamin Forehand", "Ankita", "pdehaan"]
```

Comment:
```suggestion
authors = ["Kat Anderson (@kanderson)", "Benjamin Forehand Jr. (@b4hand)", "Ankita Shrivastava (@ashrivastava)", "Peter deHaan (@pdehaan)"]
```
or maybe more generic

```suggestion
authors = ["Ecosystem Test Engineering Team"]
```

Reply:
This reminds me I need to change my github name